batch-1/movie-info/akshay/filminfo.rb

```diff
@@ -0,0 +1,88 @@
+require 'open-uri'
+require 'json'
+
+
+
+def input
+	puts"Enter the Movie title:"
+	movie_name=gets.chomp
+	movie_name=movie_name.split(" ").join("+")
+	return movie_name
```

Comment:
You don't have to write an explicit ```return``` statement in Ruby. By default, the return value of last statement in a function is returned. In your case you could end the function with just this line: ```movie_name.split(" ").join("+")```